README.md

```diff
@@ -44,24 +43,31 @@ Data Sources take two type parameters:
 </ol>
 
 ```scala
+import cats.data.NonEmptyList
+
 trait DataSource[Identity, Result]{
-  def fetch(ids: NonEmptyList[Identity]): Eval[Map[Identity, Result]]
+  def fetchOne(id: Identity): Query[Option[Result]]
+  def fetchMany(ids: NonEmptyList[Identity]): Query[Map[Identity, Result]]
 }
 ```
 
 We'll implement a dummy data source that can convert integers to strings. For convenience, we define a `fetchString` function that lifts identities (`Int` in our dummy data source) to a `Fetch`. 
```

Comment:
is just a matter of using `Fetch#apply` with an identity as shown in the `fetchString` implementation. i want to encourage people to write a little DSL (`getUser`, `getWhatever`) for their fetches since it also helps with inference.